qiskit/transpiler/passes/layout/full_ancilla_allocation.py

```diff
@@ -66,18 +66,16 @@ def run(self, dag):
             raise TranspilerError('FullAncillaAllocation pass requires property_set["layout"].')
 
         if layout:
-            FullAncillaAllocation.validate_layout(layout.get_virtual_bits(), set(dag.qubits))
-            layout_physical_qubits = list(range(max(layout.get_physical_bits()) + 1))
+            FullAncillaAllocation.validate_layout(layout._v2p, set(dag.qubits))
+            layout_physical_qubits = list(range(max(layout._p2v) + 1))
```

Comment:
Are the private accesses in this file still needed for performance? If so, can you add a comment?

Reply:
Adjusted in: https://github.com/Qiskit/qiskit-terra/pull/7036/commits/4e4da2b8d072705c4d53f7930f0866fb2115012a